ceph-volume-ansible-prs/config/definitions/ceph-volume-pr.yml

```diff
@@ -10,7 +10,7 @@
       - create
       - prepare_activate
     test_path:
```

Comment:
I think this needs to be:

    test_path: "src/ceph-volume/ceph_volume/tests/functional/lvm"